File: Linguagem C++/Estrutura Sequencial/Terreno.cpp

#include <bits/stdc++.h>

using namespace std;

int main()
{

    double larg,comp,valorMetro,valorTotal,area;

    cout << "Digite a largura do terreno: ";
    cin >> larg;
    cout << "Digite o comprimento do terreno: ";
    cin >> comp;
    cout << "Digite o valor do metro quadrado: ";
    cin >> valorMetro;

    area = larg * comp;
    valorTotal = area * valorMetro;

    cout << fixed << setprecision(2);
    cout << "Area do terreno = " << area << endl;
    cout << "Preco do terreno = " << valorTotal << endl;

    return 0;
}

File: Linguagem C#/Sequencial/Terreno.cs
using System;
using System.Globalization;

namespace Terreno {
    class Program {
        static void Main(string[] args) {

            CultureInfo CI = CultureInfo.InvariantCulture;
            double larg, comp, valorMetro, valorTerreno, area;

            Console.Write("Digite a largura do terreno: ");
            larg = double.Parse(Console.ReadLine(), CI);
            Console.Write("Digite o comprimento do terreno: ");
            comp = double.Parse(Console.ReadLine(), CI);
            Console.Write("Digite o valor do metro quadrado: ");
            valorMetro = double.Parse(Console.ReadLine(), CI);

            area = larg * comp;
            valorTerreno = area * valorMetro;

            Console.WriteLine("Area do terreno = " + area.ToString("F2", CI));
            Console.WriteLine("Preco do terreno = " + valorTerreno.ToString("F2", CI));

        }
    }
}
